code-scanning/properties/black-duck-security-scan-ci.properties.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Hi @sadmananik it looks like the linter is failing on this document. Could you remove additional white spaces?

Reply:
It should be fixed.